src/data/team.yml

```diff
@@ -12,3 +12,65 @@
         organizace brněnských akcí
     email: tomas.hejc@superkoderi.cz
     twitter: TaCcz
+-   id: 3
+    name: Michal Matuška
+    image: michal-matuska.jpg
+    position: >
```

Comment:
Can you please convert all of these in this file to single-line assignment? Position will always be a short string.

```yaml
position: organizace brněnských akcí
```